src/routes/legislative-process/+page.svelte

```diff
@@ -36,14 +38,86 @@
 						การออกกฎหมายตามรัฐธรรมนูญ 2560
 					</h2>
 					<hr />
+					<p class="body-02 mt-6">
```

Comment:
Some sentences in this paragraph, and also other paragraphs below, are bold. Please recheck with Figma.